nservicebus/operations/tuning.md

```diff
@@ -31,6 +31,8 @@ You can define a maximum value for the number of messages per second that the en
 NServiceBus will not enforce any throughput restrictions by default.
 
 
+NOTE: Currently version 6 of NServiceBus does not support `MaximumMessageThroughputPerSecond` setting.
```

Comment:
I think that the throttling will likely be dropped Particular/NServiceBus#2859. I'll remove 'currently' to avoid confusion.